src/main/flight/pid_init.c

```diff
@@ -263,6 +269,7 @@ void pidInitFeedforwardLpf(uint16_t filterCutoff, uint8_t debugAxis)
         pidRuntime.feedforwardLpfInitialized = true;
         for (int axis = FD_ROLL; axis <= FD_YAW; axis++) {
             pt3FilterInit(&pidRuntime.feedforwardPt3[axis], pt3FilterGain(filterCutoff, pidRuntime.dT));
+            pt3FilterInit(&pidRuntime.angleFeedforwardPt3[axis], pt3FilterGain(filterCutoff, pidRuntime.dT));
```

Comment:
```suggestion
            pt3FilterInit(&pidRuntime.angleFeedforwardPt3[axis], pt3FilterGain(filterCutoff, pidRuntime.level_response_gain));
```

Reply:
This is so hacky @ctzsnooze for the love of all that is good, just increase the cutoff rather than messing with the dt as this just doesn't make sense.

Reply:
You mean set the cutoff in hz in the CLI?  "For the love of all that's good"...  no-one gets Hz :-)

In principle here the intent is to control the rise and fall time of the feedforward element - to vary its time constant - so that people can experiment with fast vs slow angle feedforward.

We should not be changing the quantum of angle feedforward gain to do this.  That's because the integral of the feedforward from a stick movement over time in the 'angle' domain is... the total angle change that you get from that stick movement.  

If we halve the feedforward gain, the quad will have to fall back to the error algorithm, and the total contribution to angle achievement from feedforward will be quick, but only half what is needed.

If, on the other hand, we keep the gain the same, but vary the time constant of the filter, we can delay the rise time but get the same total effect.  

And it's the rise time that influences wobble and the 'feel' or the 'responsiveness' of the quad in angle mode.

This approach gives us the same 'area under the angle feedforward curve' while making feedforward 'fast' or 'slow'.  Since in this PR feedforward contributes 3-4 times as much to response time as the error element, controlling its temporal characteristics is important.

At this point I propose a simple value where 100 is normal and works well, whereas 200 means twice as fast (but may wobble), and 50 half as fast (but smoother), is reasonably easy for people to work with.

I'm open to alternative suggestions as to the metric for this.  For development we could use Hz or something.  

The cutoff has to be modified by dT, regardless of how we specify the cutoff.

Reply:
For a time constant for the feedforward filter control theory would suggest using 1/(Cutoff in Hz). Changing the dT in the filter gain function isn't the right approach here. It may have the desired effect but no-one reading the code will understand why a filter running at pid loop rate doesn't have the pid loop dT as the input to the filter gain function (as is the case for every other use of that function) it'll just confuse people. Just use 1/Hz as your time constant and input Hz into the filter gain.